omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml

```diff
@@ -637,16 +662,22 @@
 													<label>Condition of Baby at Discharge</label>
 													<obs conceptId="99334"  answerConceptIds="99331,162953,99332,99333,165448,165449,162954" answerLabels="MSB - Macerated Still Bath,FSB - Fresh Still Birth, NND - Immediate Neonatal Death, AL - Live Baby, AL-R - Live baby referred to another unit, AL-T - Live baby transferred to another ward/ Department, BDF - Baby Born With defect" style="dropdown" id="99334"/>
 												</div>
-												<div class="col-md-6">
+												<div class="col-md-4">
 													<label>Other Ward/Department</label>
 													<obs conceptId="165456" id="other-ward-or-department" />
 												</div>
-												<div class="col-md-6">
+												<div class="col-md-4">
 													<label>Transferred by</label>
 													<div id="transferred-by">
 														<encounterProviderAndRole encounterRole="240b26f9-dd88-4172-823d-4a8bfeb7841f" />
 													</div>
 												</div>
+												<div class="col-md-4">
+													<label>Transfered to</label>
+													<div id="transferred-to">
+														<obs conceptId="90211" id="place-transfered-to" />
+													</div>
```

Comment:
Have this field deactivated by default and activate it when ward transferred to is selected. Follow the same implementation as the previous two fields.

Reply:
@solemabrothers this one is pending